pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -616,10 +616,7 @@ func (configgen *ConfigGeneratorImpl) buildInboundFilterchains(in *plugin.InputP
 			}
 		}
 	}
-	// only inject WasmPlugins for Sidecar inbound listeners and routers outbound listeners
-	if in.Node.Type == model.SidecarProxy {
-		extension.AddWasmPluginsToMutableObjects(mutable, in.Push.WasmPlugins(in.Node))
-	}
+	extension.AddWasmPluginsToMutableObjects(mutable, in.Push.WasmPlugins(in.Node))
```

Comment:
PR title sounds like a refactor but this seems like a massive API change?

Reply:
Or is this function only called for sidecar already?

Reply:
Yeah I think that is the case.

Reply:
Tep, this is called by buildInboundFilterchains